### PR TITLE
update VuePictureSwipe.vue

### PR DESCRIPTION
**Pull request by PouriaSeyfi**
some times we want to show only first thumbnail. 
the "SingleThumb" prob help us to do that.

### CODE REVIEW

File: src/VuePictureSwipe.vue [hunk 1]
```diff
@@ -3,6 +3,7 @@
     <div class="my-gallery" itemscope itemtype="http://schema.org/ImageGallery">
 
       <figure
+          v-show="index > 0 && !singleThumb"
```
Comment (PouriaSeyfi) [inline src/VuePictureSwipe.vue]: hide other thimbnails except first one

Comment (rap2hpoutre) [inline src/VuePictureSwipe.vue]: Maybe we could try to be more explicit (by replacing `singleThumb` by `singleThumbnail`:

```suggestion
          v-show="index > 0 && !singleThumbnail"
```

File: src/VuePictureSwipe.vue [hunk 2]
```diff
@@ -89,6 +90,10 @@
       options: {
         default: () => ({}),
         type: Object
+      },
+      singleThumb :{
```
Comment (rap2hpoutre) [inline src/VuePictureSwipe.vue]:
```suggestion
      singleThumbnail :{
```

**Review by rap2hpoutre — COMMENTED** [review]
Thank you for your contribution! 🎉  Just one suggestion about variable name and it's good to go (see comments below)

**Comment by PouriaSeyfi** [comment]
thank you

**Pull request closed without merging**